plugins/modules/database/saphana/hana_query.py

```diff
@@ -21,13 +22,17 @@
         type: str
         required: true
     user:
-        description: A dedicated username. Defaults to C(SYSTEM).
+        description: A dedicated username. The user could be also in hdbuserstore. Defaults to C(SYSTEM).
         type: str
         default: SYSTEM
+    userstore:
+        description: If C(true) the user must be in hdbuserstore.
+        type: bool
+        default: false
+        version_added: 3.5.0
     password:
         description: The password to connect to the database.
```

Comment:
```suggestion
        description:
          - The password to connect to the database.
          - "B(Note:) Since the passwords have to be passed as command line arguments, I(userstore=true) should
            be used whenever possible, as command line arguments can be seen by other users
            on the same machine."
```

Reply:
Thank you